ext/curl/curl.stub.php

```diff
@@ -3553,12 +3553,80 @@
  */
 const CURLOPT_SAFE_UPLOAD = UNKNOWN;
 
+/**
+ * @strict-properties
+ */
+class CurlException extends \Exception {};
+
+/**
+ * @strict-properties
+ */
+class CurlHandleException extends \CurlException {};
+
+/**
+ * @strict-properties
+ */
+class CurlMultiException extends \CurlException {};
+
+/**
+ * @strict-properties
+ */
+class CurlShareException extends \CurlException {};
+
 /**
  * @strict-properties
  * @not-serializable
  */
 final class CurlHandle
 {
+	public function __construct(?string $uri = null) {}
+
+	/** @alias curl_errno */
+	public function errno(): int {}
+
+	/** @alias curl_error */
+	public function error(): string {}
+
+	/** @alias curl_strerror */
+	static public function strerror(int $code): string {}
+
+	/** @alias curl_escape */
+	public function escape(string $str): string {}
+
+	/** @alias curl_unescape */
+	public function unescape(string $str): string {}
+
+	/**
+	 * If CURLOPT_RETURNTRANSFER is True, returns the result on success,
```

Comment:
I know this is directly specifying current behaviour, but I wish the return wasn't ambigious based on the current option.